client/src/components/files/FileListItem.vue

```diff
@@ -118,11 +122,11 @@ defineExpose({
   props,
 });
 
-function getFileIcon(): string {
+function fileIcon(): string {
```

Comment:
I would update `getFileIcon()` to take a boolean (add an arg `isDir = false`, something like this) and return the correct icon directly.